packages/main/test/pages/InputFieldMinWidth.html

```diff
@@ -1,13 +1,27 @@
 <!DOCTYPE html>
 <html>
+
 <head>
 	<meta charset="UTF-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
You don't need to add this back. See #5663

Reply:
vs code autocomplete :D i will change it